package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@latitude-data/promptl",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
After rebasing with main it will be 0.3.2

Reply:
Did you publish the new version yet? We can release both changes in the same version

Reply:
yep i did